### ISSUE TITLE
F/A-18C Hornet: Missing IFEI_R0_TEXTURE

### ISSUE THREAD

**Issue by charliefoxtwo** [DCSFlightpanels/dcs-bios#46]
The hornet has L0, L50, L100, R50, and R100 texture outputs, but no R0 texture output.

Side note - do you know which bios code should be used to get the nozzle position on the IFEI? I couldn't find it anywhere. I've been using `EXT_NOZZLE_POS` but I'm not sure if that's exactly what's on the IFEI (e.g. at very low nozzle positions like 3%, the IFEI reads 0% - but that may just be a resolution thing).

Also potentially missing: 
  - The fuel_up/down gauges should have an L/R texture as well I believe, which I couldn't find either.
  - In Time Set Mode on the IFEI, the output corresponding to `txt_T` is missing

**Comment by WarLord211** [comment]
R0 Fixed was a typo

Nozzle export we got never to work. Aus Workaround i created the EXT_Nozzle.

*fuel_up/down gauges should have an L/R texture  -- I must say i dont know what or how this code works. 
*Time Set Mode on the IFEI, the output corresponding to txt_T is missing -- i dont understand

**Comment by charliefoxtwo** [comment]
Basically, a new bios output needs to be added called `IFEI_T`. It is a 6 character string that goes in the same location in the jet as `IFEI_FUEL_UP`. It is defined in the hornet luas as `txt_T`. If you're having trouble locating it, it should be right next to `txt_TimeSetMode`. I don't remember exactly which lua file it's in, but it should be the ifei lua related to the time set page.

Let me know if this is still unclear.

**Comment by WarLord211** [comment]
ok here a test. (the zip contains the lua)

override the F18 lua in lib folder
start a flight
after that you can program or verifie it via refpage.

i cant test it by my self because i have no idea how to operate the F-18

what is this "T"? (for o good name for it)
[FA-18C_hornet.zip](https://github.com/DCSFlightpanels/dcs-bios/files/6379114/FA-18C_hornet.zip)

**Comment by charliefoxtwo** [comment]
`IFEI_T` shows `T` when setting the time on the IFEI, `D` when setting the date on the IFEI, and the time zone offset when adjusting the local time zone (in the form of something like `4 +   `) on the IFEI.

Just tested, and seems to work as expected. I've mapped pretty much everything on the IFEI and as far as I can tell, no regressions. Thanks!

For the fuel up down l/r texture, I would _think_ it's similar to the L0/R0 texture, but when I played around with it yesterday I think it resulted in other things breaking. I'll toy with it some more.
